### ISSUE TITLE
'Show zones beginning with:' unavailable/greyed out

### ISSUE THREAD

**Issue by IroHaturo** [poweradmin/poweradmin#435]
### Discussed in https://github.com/poweradmin/poweradmin/discussions/433

<div type='discussions-op-text'>

<sup>Originally posted by **IroHaturo** May 22, 2022</sup>
In list_zones.php I cannot click the letters, that usually would display zones starting w. a certain letter, see screenshots:

Initially [a] is active and [show all]
<img width="475" alt="Screenshot 2022-05-22 at 16 01 57" src="https://user-images.githubusercontent.com/57433686/169700162-42c5ca17-802c-46f6-9c22-e74b0cb1a2f6.png">

with [show all] clicked:
<img width="581" alt="Screenshot 2022-05-22 at 15 51 23" src="https://user-images.githubusercontent.com/57433686/169699215-38286b86-06c6-4c6d-b221-34c82608b0eb.png">

Same w. diff. browsers, so no cookie/cache issue

After gratefully using PDNS w. Poweradmin since many years, I just installed a phresh server from scratch, Debian 10 w. poweradmin v2.2.3, mysql  Ver 15.1 Distrib 10.3.34-MariaDB and PowerDNS Authoritative Server 4.1.6.
Things work fine, except the lil issue above.

Keep up the good work!
Cheers,
~iro
</div>

**Comment by RicardovanTongeren** [comment]
I seem to have the same issue. Did some debugging and found out the following.

on /inc/pagination.inc.php on line 129 there is a WHERE statement that won't return anything because our zones table is empty.
```
    $query = "SELECT
                        DISTINCT ".dbfunc_substr()."(domains.name, 1, 1) AS letter
                        FROM domains
                        LEFT JOIN zones ON domains.id = zones.domain_id";
//                      LEFT JOIN zones ON domains.id = zones.domain_id
//                      WHERE zones.owner = " . $userid;
    $db->setLimit(36);
```

By removing the WHERE statement i can use the pagination again but this is not a solution.

Not sure how we ended up in this. Maybe someone knows the official fix.

**Comment by bnchdan** [comment]
This is because you have zones that don t have owners. You should set an owner to all zones. For example, you can run 
`insert into zones (domain_id, owner, zone_templ_id) select id,1,0 from domains ` 
to set all zones to the owner with id 1
` select * from users \G`
`*************************** 1. row ***************************`
`         id: 1`
`   username: admin`
`   password: `
 `  fullname: Administrator`
 `     email: admin@example.net`
`description: Administrator with full rights.`
` perm_templ: 1`
 `    active: 1`
`   use_ldap: 0`
`*************************** 2. row ***************************`
or you can set from web interface
![image](https://user-images.githubusercontent.com/30780133/175517458-4b325941-9596-4bce-a770-0d1faf680b2c.png)

@RicardovanTongeren if you modified that and you will have a user with domains: abc.com and bbc.com the 'Show zones beginning with:'  will be wrong.

**Comment by edmondas** [comment]
Fixed by #436
